Cargo.toml

```diff
@@ -1,14 +1,55 @@
 [workspace]
-
+resolver = "2"
 members = [
     "lib/srv/desktop/rdp/rdpclient",
     "web/packages/teleport/src/ironrdp"
 ]
 
+[workspace.package]
+edition = "2021"
+license = "AGPL-3.0-only"
+homepage = "https://github.com/gravitational/teleport"
+repository = "https://github.com/gravitational/teleport"
+authors = [
+    "Teleport <goteleport.com>",
+    "Andrew Lytvynov <andrew@awly.dev>",
+    "Zac Bergquist <zac@goteleport.com>",
+    "Isaiah Becker-Mayer <isaiah@goteleport.com>",
+    "Przemko Robakowski <przemko.robakowski@goteleport.com>"
+]
+keywords = ["teleport", "rdp", "desktop"]
+publish = false
+
 [profile.dev]
 debug = 1
 lto = "off"
 
 [profile.release]
 debug = 1
 codegen-units = 1
+
+[workspace.dependencies]
+byteorder = "1.5.0"
+bytes = "1.4.0"
+ironrdp-cliprdr = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
+ironrdp-connector = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
+ironrdp-graphics = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
+ironrdp-pdu = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
+ironrdp-rdpdr = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
+ironrdp-rdpsnd = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
+ironrdp-session = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
+ironrdp-svc = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
+ironrdp-tls = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
+ironrdp-tokio = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
+# Uncomment the following lines to use local crates instead of the ones from github
```

Comment:
Do we need to keep this in here? Feels a bit sloppy leaving commented out stuff present.

Reply:
The reason I want to leave some breadcrumbs is that unlike most crates that I've worked with, the translation to a local crate isn't a straightforward mapping from 
```
ironrdp-cliprdr = { git = "https://github.com/Devolutions/IronRDP", rev = "6588f7cdc71cbaa24ae6d18b7f8d0d3ed48c1377" }
```

to

```
ironrdp-cliprdr = { path = "/path/to/local/IronRDP" }
```

rather it's 

```
ironrdp-cliprdr = { path = "/path/to/local/IronRDP/crates/ironrdp-cliprdr" }
```

(you need that additional `/crates/ironrdp-cliprdr`). 

I've changed it to a small reminder comment
```
# Note: To use a local IronRDP repository as a crate (for example, ironrdp-cliprdr), define the dependency as follows:
# ironrdp-cliprdr = { path = "/path/to/local/IronRDP/crates/ironrdp-cliprdr" }
```